### PR TITLE
Give Sectopods Resilience

### PR DESCRIPTION
**Pull request by Grobobobo**
As Discussed. I think.

### CODE REVIEW

**Comment by BlackRose39** [comment]
I agree, and also, give them the "easytohit," since they're big targets.

**Comment by Grobobobo** [comment]
That will depend on PaledBrook.

**Comment by ansabalirai** [comment]
As commented on github, I would suggest linking it to a DE instead and applied to a configurable percentage. This would also make it possible to extend it to other massive enemies, e.g. cyber ball.

**Comment by Grobobobo** [comment]
The point is to make them more resistant to Kubikuri, so making it a DE doesn't make much sense.

File: LongWarOfTheChosen/Src/LW_Overhaul/Classes/LWTemplateMods.uc [hunk 1]
```diff
@@ -1945,6 +1945,9 @@ function GeneralCharacterMod(X2CharacterTemplate Template, int Difficulty)
 			Template.Abilities.AddItem('CoupdeGrace2');
 			Template.Abilities.AddItem('Whirlwind2');
 			break;
+		case 'Sectopod'
+			Template.Abilities.AddItem('Resilience');
```
Comment (pledbrook) [inline LongWarOfTheChosen/Src/LW_Overhaul/Classes/LWTemplateMods.uc]: You're missing a `break;` here.

**Comment by pledbrook** [comment]
I'm tempted to give all robotics a slightly weaker Resilience, just to make them even more distinctive. That would be my ultimate goal: make enemy units as distinctive as possible, with significant strengths and significant weaknesses. Ideally with multiple ways to deal with them.

**Comment by Grobobobo** [comment]
I fixed that in the second commit. I guess that's why they should be informative, huh?

**Comment by pledbrook** [comment]
I would be OK with an easy-to-hit mod, or a negative defence. Not sure about giving them 'CivilianEasyToHit' though :smile:

**Comment by Grobobobo** [comment]
I don't even think civilians should have 'CivilianEasyToHit'.

**Comment by ansabalirai** [comment]
Well robots already do not have the normal 40% flank percentage that you get on other aliens,, so... bluescreen strat incoming?

**Comment by pledbrook** [comment]
I think the current Resilience may be too strong, and it's worth bearing in mind that the Stocks will be nerfed too, so stacking crit is a bit harder.

Regardless, there are still several anti-robotic options, lots of shred (although coilgun shred needs fixing), plus several sources of hack defence reduction with shutdown, etc. Let's see how it plays out.

**Pull request merged**